lib/json.dl

```diff
@@ -164,3 +164,27 @@ typedef JsonNum = // Integer number.  The value must be in the range between the
                 | // NaN and infinity are not valid JSON values.
                   // When serializing to JSON, such values will be replaced with 0.
                   JsonFloat{d: double}
+
+/* Wrapper type that serializes its inner object into binary formats via JSON string.
```

Comment:
I don't really understand the connection between JSON, strings, and binary serialization.